gui.py

```diff
@@ -2046,15 +2046,15 @@ def __init__(self, twitch: Twitch):
         )
         # remove Notebook.focus from the Notebook.Tab layout tree to avoid an ugly dotted line
         # on tab selection. We fold the Notebook.focus children into Notebook.padding children.
-        if theme != "classic":
+        if theme != "classic" and sys.platform != "darwin":  # Skip on macOS
```

Comment:
I'm unable to create a suggestion through Github that's big enough to encompass my actual suggestion here, and apparently it's telling me that "suggestions for deleted lines are not supported", even though there's no deletions here. Either way, I'll paste in my suggestion here, and it'll be up for you to paste it into the PR, and then push it - the code should land right into L2047-2063 range.

That is if you want the commit and contribution to be fully yours. Alternatively, I can just push this out on my end, with a commit where I'd credit you in the extended description. Up to you to decide.

```py
        # add padding to the tab names
        style.configure("TNotebook.Tab", padding=[8, 4])
        if theme != "classic" and sys.platform != "darwin":  # Skip these for classic theme or macOS
            # remove Notebook.focus from the Notebook.Tab layout tree to avoid an ugly dotted line
            # on tab selection. We fold the Notebook.focus children into Notebook.padding children.
            original = style.layout("TNotebook.Tab")
            sublayout = original[0][1]["children"][0][1]
            sublayout["children"] = sublayout["children"][0][1]["children"]
            style.layout("TNotebook.Tab", original)
            # remove Checkbutton.focus dotted line from checkbuttons
            style.configure("TCheckbutton", padding=0)
            original = style.layout("TCheckbutton")
            sublayout = original[0][1]["children"]
            sublayout[1] = sublayout[1][1]["children"][0]
            del original[0][1]["children"][1]
            style.layout("TCheckbutton", original)
```

Reply:
Replaced with you code, thanks.

Reply:
Yeah, my bad, read it wrong. Fixed.